test/e2e/resources/record_set_simple.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: route53.services.k8s.aws/v1alpha1
+kind: RecordSet
+metadata:
+  name: $SIMPLE_RECORD_NAME
+spec:
+  name: $SIMPLE_RECORD_DNS_NAME
+  hostedZoneID: $HOSTED_ZONE_ID
+  recordType: A
+  resourceRecords:
+  - value: $IP_ADDR
+  tTL: 300
```

Comment:
i believe this should be `ttl`

Reply:
i will send a patch to `github.com/aws-controllers/pkg` and `code-generator`. Need to hold until we fix this :)
/hold